format-proposal.md

```diff
@@ -60,6 +60,18 @@ var int : a = 1;
 variable_type("a",int).
 variable_value("a",value,1).
 
+% FZN
```

Comment:
Regarding you comment
> I still do get the output
> variable_type("X_INTRODUCED_6_",int_in_range(0,7773)).
> while int_in_range is never used in the format proposal.

I added these test cases. But I'm not sure it is correct. maybe it should be 
`variable_type("a",int,range,(value,1,value,3))).` ?